OrbitQt/servicedeploymanager.cpp

```diff
@@ -335,6 +337,23 @@ outcome::result<void> ServiceDeployManager::ConnectToServer() {
   return outcome::success();
 }
 
+void ServiceDeployManager::StartWatchdog() {
+  orbit_service_task_->Write(kSshWatchdogPassphrase);
+
+  // ssh_watchdog_timer_ = QPointer(new QTimer());
+
+  QObject::connect(&ssh_watchdog_timer_, &QTimer::timeout, [this]() {
+    if (orbit_service_task_) {
+      orbit_service_task_->Write(".");
+    } else {
+      ssh_watchdog_timer_.stop();
+    }
+  });
+
+  ssh_watchdog_timer_.setInterval(kSshWatchdogInterval);
+  ssh_watchdog_timer_.start();
```

Comment:
Nit: If you're removing the comment anyway, you could also make one line out of this. ssh_watchdog_timer_.start(kSshWatchdogInterval). But not important at all. If your prefer the two line version it's fine as well.

Reply:
I actually missed that overload in the documentation. Its split up in signals and normal member functions. Anyway, sloppy again on my side